.changeset/little-steaks-rule.md

```diff
@@ -0,0 +1,17 @@
+---
+'astro': patch
+---
+
+Removing non-stable SVG features in preparation for releasing stable version.
+
+Added:
+
+- `SVGComponent` type for easier passing of SVGs
```

Comment:
Does this affect how people *use* the feature, or is this an internal implementation?

If internal, and not something people write out in their code, probably not necessary to mention here/in docs. 

However, you can describe this as a DX feature (if e.g. it helps with autocomplete or something?):

This release adds a new `SVGComponent` type for... (and whatever the advantage is).

Reply:
I'm going to remove this as it seems to be an incorrect usage.